data/json/items/armor/holster.json

```
@@ -321,6 +321,47 @@
       }
     ]
   },
+  {
+    "id": "shoulder_holster",
+    "copy-from": "holster",
+    "type": "ARMOR",
+    "name": { "str": "shoulder holster" },
+    "description": "A comfortable quick draw holster for larger pistols.  One side holsters a gun and the other side holds two magazines.  Activate to holster/draw a gun.",
+    "price_postapoc": 500,
+    "material": [ "leather" ],
+    "pocket_data": [
+      {
+        "magazine_well": "200 ml",
+        "pocket_type": "CONTAINER",
+        "holster": true,
+        "min_item_volume": "100 ml",
+        "max_contains_volume": "650 ml",
+        "max_contains_weight": "2 kg",
+        "max_item_length": "250 mm",
+        "moves": 30
+      },
+      {
+        "pocket_type": "CONTAINER",
+        "max_item_length": "240 mm",
+        "max_contains_volume": "750 ml",
+        "max_contains_weight": "2 kg",
+        "moves": 40,
+        "flag_restriction": [ "MAG_COMPACT" ],
+        "volume_encumber_modifier": 0.3
+      },
+      {
+        "pocket_type": "CONTAINER",
+        "max_item_length": "240 mm",
+        "max_contains_volume": "750 ml",
+        "max_contains_weight": "2 kg",
+        "moves": 40,
+        "flag_restriction": [ "MAG_COMPACT" ],
+        "volume_encumber_modifier": 0.3
+      }
+    ],
+    "use_action": { "type": "holster" },
+    "armor": [ { "encumbrance": [ 1, 4 ], "coverage": 5, "covers": [ "torso_upper" ] } ]
```

if you are gonna do the new format with volume_encumber_modifier may as well just standardize it, most pouches like this have a .3 - .35 volume encumber modifier. If you don't put an upper bound it'll just hand encumbrance itself.
```suggestion
    "armor": [ { "encumbrance": 1, "coverage": 5, "covers": [ "torso_upper" ] } ]
```